src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/ubuntu-dotnet-local-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Argument = -d destdir
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script installs dotnet cli on the linux machine
+
+OPTIONS:
+   -h      Show this message
+   -d      directory where dotnet cli will be installed
+
+EOF
+}
+
+DESTDIR=
+while getopts “hd:” OPTION
+do
+     case $OPTION in
+         h)
+             usage
+             exit 1
+             ;;
+         d)
+             DESTDIR=$OPTARG
+             ;;
+         ?)
+             usage
+             exit
+             ;;
+     esac
+done
+
+echo $DESTDIR
+if [[ -z $DESTDIR ]]
+then
+     usage
+     exit 1
+fi
+
+echo "Initiating local dotnet install"
+wget https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh
```

Comment:
To the dotnet-install script I am explicitly passing the version number which in turn is set in our xunit-performance repo in the DotNetCliVersion.txt file, I believe that should ensure control over versions of dotnet cli that will be installed.